TheForceEngine/TFE_FrontEndUI/modLoader.cpp

```diff
@@ -90,7 +90,8 @@ namespace TFE_FrontEndUI
 
 	void modLoader_read()
 	{
-		if (s_modsRead) { return; }
+		// Reuse the cached mods unless no mods have been read yet.
+		if (s_modsRead and s_mods.size() > 0) { return; }
```

Comment:
I'm surprised this works, it should be '&&' instead of 'and'.

Reply:
It 100% works. Apparently they are alternatives https://stackoverflow.com/questions/47174920/difference-between-and-and-operators . I will switch to && for consistency (I am coming from a python background).

Reply:
C# has implemented "and" and "or"  as part of its pattern matching 

Instead of `if (a == b || a == c)`
you can now do `if (a is b or c)`

I still find it disconcerting...

Reply:
Oh that's neat - I love it.